doc/nrf/ug_nrf52.rst

```diff
@@ -268,6 +267,29 @@ To perform a FOTA upgrade, complete the following steps:
       .. note::
          There is currently no support for the FOTA process in nRF Connect for Desktop.
 
+FOTA upgrade sample
+===================
+
+The :ref:`zephyr:smp_svr_sample` demonstrates how to set up your project to support FOTA upgrades.
+
+The sample documentation is derived from the Zephyr project and is incompatible with the :ref:`ug_multi_image`.
+When working in the nRF Connect SDK environment, you can ignore the part of the sample documentation which describes the building and image flashing steps.
+Due to the :ref:`ug_multi_image` feature, you can build and flash the :ref:`zephyr:smp_svr_sample` as any other sample using the following commands:
+
+.. code-block:: console
+
+    west build -b *build_target* -- -DOVERLAY_CONFIG=overlay-bt.conf
+    west flash
+
+Please remember to indicate the :file:`overlay-bt.conf` overlay configuration for the Bluetooth transport like in the command example above.
+This configuration was carefully selected to achieve the maximum possible throughput of the FOTA upgrade transport over Bluetooth with the help of the following features:
+
+- Bluetooth MTU: to increase the packet size of a single Bluetooth packet transmitted over the air (:kconfig:option:`CONFIG_BT_BUF_ACL_RX_SIZE` and others).
+- Bluetooth Connection Parameters: to change the connection interval and latency when the SMP service is being used (:kconfig:option:`CONFIG_MCUMGR_SMP_BT_CONN_PARAM_CONTROL`).
```

Comment:
```suggestion
- Bluetooth connection parameters: to change the connection interval and latency when using the SMP service (:kconfig:option:`CONFIG_MCUMGR_SMP_BT_CONN_PARAM_CONTROL`).
```

Reply:
Please check it again. I would like to rephrase because I realized that my old phrasing did not convey my message good enough.